packages/main/src/Input.js

```diff
@@ -573,7 +581,7 @@ class Input extends UI5Element {
 			- value of the host and the internal input should be differnt in case of actual input
 			- input is called when a key is pressed => keyup should not be called yet
 		*/
-		const skipFiring = (this.getInputDOMRef().value === this.value) && isIE() && !this._keyDown && !!this.placeholder;
+		const skipFiring = (this.inputDomRef.value === this.value) && isIE() && !this._keyDown && !!this.placeholder;
```

Comment:
I see that in a lot of places in the code this.inputDomRef is now used and to work as expected, the code assumes the _onfocusin method has been called to await this.getInputDOMRef() to have  this.inputDomRef properly assigned. 

It might be working now, might not be working in some cases, we did not find yet, but I find it shaky to be honest. To make the code more robust I would go for:

```js
const inputDOMref =  await this.getInputDOMRef();
inputDOMref.value
or
(await this.getInputDOMRef()).value;
```